### ISSUE TITLE
New reforged W3I format

### ISSUE THREAD

**Issue by Frotty** [inwc3/wc3libs#48]
```
2019-11-17 10:09:06 [Wurst LanguageWorker] WARN  languageServer - Exception occurred
java.lang.RuntimeException:
unknown format 31
supported formats:
-1
28
25
18
    at de.peeeq.wurstio.languageserver.ProjectConfigBuilder.apply(ProjectConfigBuilder.java:83)
    at de.peeeq.wurstio.languageserver.requests.RunMap.execute(RunMap.java:123)
    at de.peeeq.wurstio.languageserver.requests.UserRequest.run(UserRequest.java:31)
    at de.peeeq.wurstio.languageserver.LanguageWorker.lambda$getNextWorkItem$1(LanguageWorker.java:171)
    at de.peeeq.wurstio.languageserver.LanguageWorker$Workitem.run(LanguageWorker.java:29)
    at de.peeeq.wurstio.languageserver.LanguageWorker.run(LanguageWorker.java:146)
    at java.base/java.lang.Thread.run(Thread.java:830)
Caused by: net.moonlightflower.wc3libs.bin.BinStream$StreamException: null
    at net.moonlightflower.wc3libs.bin.Wc3BinInputStream.getFormat(Wc3BinInputStream.java:428)
    at net.moonlightflower.wc3libs.bin.app.W3I.read_auto(W3I.java:2552)
    at net.moonlightflower.wc3libs.bin.app.W3I.read(W3I.java:2558)
    at net.moonlightflower.wc3libs.bin.app.W3I.read(W3I.java:2601)
    at net.moonlightflower.wc3libs.bin.app.W3I.<init>(W3I.java:2625)
    at de.peeeq.wurstio.languageserver.ProjectConfigBuilder.prepareW3I(ProjectConfigBuilder.java:91)
    at de.peeeq.wurstio.languageserver.ProjectConfigBuilder.apply(ProjectConfigBuilder.java:40)
    ... 6 common frames omitted
```

see https://github.com/wurstscript/WurstScript/issues/906

**Comment by Frotty** [comment]
W3I file example: https://github.com/wurstscript/WurstScript/files/3855436/war3map.zip

**Comment by WaterKnight** [comment]
Implemented in 966bf25610e673bd2b0554c3e01d58c9c0803120, script injection now requires to state the target game version,
